### DISCUSSION TITLE
How can add an image tag?

### DISCUSSION THREAD

**Discussion by rmazur** [podman-desktop/podman-desktop#764] [General]
After pull a new image from a registry I like to change image tag and use it within containerfile. How could I do that with Podman desktop?

**Comment by benoitf** [comment]
Hello, there is no tag operation on an existing image.
Is it something you have using other tools ?

workaround could be do build a new image just using the image that you want to tag

Containerfile being
```
FROM my-existing-image
```
and you specify the new name of your image when building it (like `my-new-image:my-tag`)
